pep-0582.rst

```diff
@@ -177,6 +187,9 @@ For example, if we execute ``python /usr/share/myproject/fancy.py`` from the
 ``/usr/share/myproject/`` directory, it will be used. Any potential
 ``__pypackages__`` directory in ``/tmp`` will be ignored.
 
+This also means we will not scan any parent directory while executing any
+script. If we want to execute scripts inside of ``~/bin/`` directory, then
+the ``__pypackages__`` directory must be inside of ``~/bin/`` directory.
```

Comment:
```suggestion
This also means we will not scan any parent directory while executing scripts.
If we want to execute scripts inside of the ``~/bin/`` directory, then
the ``__pypackages__`` directory must be inside of the ``~/bin/`` directory.
```

Fix a couple grammar errors (missing definite articles) and clarify phrasing/remove repeated word.